lib/node-sass-import.js

```diff
@@ -35,14 +41,31 @@ var resolver = function (url, baseDir, done) {
       }
     }
   }, function (err, file) {
-    if (err) throw err;
+    if (err) return done(err);
 
     // Strip the extension so .css gets evaluated properly as .scss
     file = file.replace(/\.(sa|sc|c)ss$/, '');
-    done({ file: file });
+    done(null, file);
```

Comment:
I had to change how `resolver()` works. It used to call `done(result)`, but now it calls `done(null, result)` to make a Node-style callback function. This allows us to use [async](https://www.npmjs.com/package/async).parallel() on it.